primer.md

```diff
@@ -145,6 +145,12 @@ user-stories to explain the rationale and need for them. This supporting
 information will be added to the [Prior Art](#prior-art) section of this
 document.
 
+Extension attributes should be kept minimal to ensure the CloudEvent can be
```

Comment:
Propose small clarifying change:

Extension attributes required by a single CloudEvent should be kept minimal

e.g. It's ok if the community as a whole as lots of extensions for different domains, but in practice few would be needed for an individual event than needs to be transported on the wire

Reply:
What about:
Event producers should consider the technical limitations that might be encountered when adding extensions to a CloudEvent. For example, the [HTTP Binary Mode](http-transport-binding.md#31-binary-content-mode) uses HTTP headers to transport metadata; most HTTP servers will reject requests with excessive HTTP header data, with limits as low as 8kb. Therefore, the aggregate size and number of extension attributes should be kept minimal.